### ISSUE TITLE
Got error when trying to create a test for a record

### ISSUE THREAD

**Issue by 304NotModified** [mattwhitfield/unittestgenerator#55]
```c#
namespace MyNameSpace
{
    public record MyRecord
    {
        public string Method1()
        {
            return "Hai";
        }
    }
}
```

![image](https://user-images.githubusercontent.com/5808377/155214712-b520211a-ab1c-4abc-aa5a-c2570eb50bb7.png)

gives:

![image](https://user-images.githubusercontent.com/5808377/155214802-8d64bec7-ebdd-4cbc-84e1-f3712aa1f21d.png)

note: it works when MyRecord is a `class`

- VS2022 (17.1.0)
- Extension: 0.100.0.0
- .NET 5 class library

**Comment by 304NotModified** [comment]
I guess this is the error line:

https://github.com/mattwhitfield/unittestgenerator/blob/3b988a1e2f4f55dfe86bb106038881fa644501f5/src/Unitverse/Helper/CodeGenerator.cs#L59

**Comment by mattwhitfield** [comment]
Are you generating by right clicking on the file and selecting 'generate tests'? Does the same happen if you right click on the type name in the editor and click 'generate tests'?

**Comment by mattwhitfield** [comment]
Actually, nvm. It doesn't give you the option on the type name because I'm an idiot 🤣

Fix coming

**Comment by mattwhitfield** [comment]
Should be fixed by 0.101 - apologies

**Comment by 304NotModified** [comment]
> apologies

No need! Thanks for the excellent work :)
